docs/competition/README.md

```diff
@@ -0,0 +1,12 @@
+# Competition Event
+
+The culmination of Student Robotics annual [Competition Programme][programme] is
+the the Competition Event. This is typically a two-day event taking place on a
+weekend in April.
+
+During the course of the competition event the teams' robots compete in a large
+number of [matches](./matches/README.md) in order to determine the overall winner.
+
+This section outlines the running of the competition event.
+
+[programme]: ../programme/README.md
```

Comment:
Any reason to have this split out of the main text? Makes sense if it's used multiple times, but less so as it's just used once. Would be good to keep things consistent

Reply:
Mostly for nice text wrapping -- this would have created odd looking line lengths on the line it's in.